wurst/file/Base64.wurst

```diff
@@ -0,0 +1,270 @@
+package Base64
+
+import Bitwise
+import StringUtils
+import ByteBuffer
+import ChunkedString
+import Execute
+import ErrorHandling
+
+/*  Base64 package provides an encoder/decoder pair that allows serializing and deserializing of any data.
```

Comment:
Phrasing: encoder/decoder pair sounds like the library provides function to emit a linked encoder-decoder pair like [channel](https://doc.rust-lang.org/std/sync/mpsc/fn.channel.html). Better: "provides encoding and decoding function for the common base64 format"

Phrasing: "serializing and deserializing of any data" - what does *any data* mean? Wurst is a strongly typed language: can you rephrase this to explain the type relationship?

Reply:
Attempted to make it better.

Reply:
The first change looks good, but what does `bytes` mean? Jass doesn't have a `byte` type